src/utils/models/index.ts

```diff
@@ -27,3 +27,15 @@ export const QuickStartModel: K8sModel = {
   plural: 'consolequickstarts',
   propagationPolicy: 'Background',
 };
+
+export const ManagedClusterModel: K8sModel = {
```

Comment:
If this is specifically related to ACM I suggest moving it to the multicluster directory.